.circleci/config.yml

```diff
@@ -241,9 +241,9 @@ workflows:
   build-and-test:
     jobs:
       - lint-testunit
-      # - e2e-test:
-      #     requires:
-      #       - lint-testunit
+      - e2e-test:
```

Comment:
Tests take to long to run.
Add a hold step similar to `ios-hold-testflight` and then we can choose when to run them.